uconnrcmpy/conditions.py

```diff
@@ -347,7 +347,11 @@ def change_EOC_time(self, experiment, time, is_reactive=True):
             if str(experiment) in self.reactive_experiments:
                 experiment = self.reactive_experiments[str(experiment)]
             elif str(experiment) in self.nonreactive_experiments:
-                experiment = self.nonreactive_experiments[str(experiment)]
+                exp = Experiment(experiment, cti_file=self.cti_file)
```

Comment:
You can simplify this by writing `experiment = Experiment...` and deleting `experiment = self.reactive_experiments...`